src/main/java/uk/co/ramp/covid/simulation/output/DailyStats2.java

```diff
@@ -0,0 +1,203 @@
+        
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) return true;
+            if (o == null || getClass() != o.getClass()) return false;
+            IntValue other = (IntValue) o;
+            return value == other.value;
+        }
+
+        @Override
+        public int hashCode() {
+            return value;
+        }
+    }
+    
+    public class DoubleValue extends Value {
+        private double realValue;
+        
+        public DoubleValue(String csvHeader) {
+            super(csvHeader);
+        }
+        
+        public void set(double value) { this.realValue = value; }
+
+        @Override
+        public String toString() { return Double.toString(realValue); }
+        
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) return true;
+            if (o == null || getClass() != o.getClass()) return false;
+            DoubleValue other = (DoubleValue) o;
+            return realValue == other.realValue;
+        }
+
+        @Override
+        public int hashCode() {
+            return Double.hashCode(realValue);
+        }        
+    }
+    
+    private IntValue add(String csvHeader) { return new IntValue(csvHeader); }
+    
+    public DailyStats2(Time t) {
+        day.set(t.getAbsDay());
+    }
+    
+    public void processPerson(Person p) {
+        switch (p.cStatus()) {
+            case HEALTHY: healthy.increment(); break;
+            case LATENT: exposed.increment(); break;
+            case ASYMPTOMATIC: asymptomatic.increment(); break;
+            case PHASE1: phase1.increment(); break;
+            case PHASE2: phase2.increment(); break;
+            case RECOVERED: recovered.increment(); break;
+            case DEAD: dead.increment(); break;
+        }
+        
+        if (p.isHospitalised()) {
+            inHospital.increment();
+        }
+    }
+    
+    private int sum(List<IntValue> stats) {
+        return stats.stream().mapToInt(s -> s.value).sum();
+    }
+
+    public int getTotalPopulation() {
+        return sum(totalPopulation);
+    }
+
+    public int getTotalInfected() {
+        return sum(totalInfected);
+    }
+    
+    public int getTotalDailyInfections() {
+        return sum(totalDailyInfections);
+    }
+   
+    public String logString() {
+        return String.join(" ", logged.stream().map(s -> s.logString()).toArray(String[]::new));
+    }
+    
+    public void log(){
+        LOGGER.info(logString());
+        //LOGGER.info("Day = {} Healthy = {} Latent = {} Asymptomatic = {} Phase 1 = {} " +
+        //                "Phase 2 = {} Hospitalised = {} Dead = {} Recovered = {}",
+        //        day, healthy, exposed, asymptomatic,phase1, phase2, inHospital, dead, recovered);
+    }
+
+    // TODO: We should probably either move the outputCSV method in here,
+    // or alternatively return the list of values (and not pass in the CSVPrinter here)
```

Comment:
We (Blair and I) prefer the second option suggested here; return the list of values.